list2/ex3/index.js

```diff
@@ -0,0 +1,24 @@
+import promptSync from 'prompt-sync';
+const prompt = promptSync();
+
+const DISTANCE_TO_THE_DISCOUNT = 200;
+const PRICE_WITH_DISCOUNT = 0.45;
+const PRICE_WITHOUT_DISCOUNT = 0.50;
+
+export function travelCostCalculation() {
+    console.log('How many km will you travel? (use an INTEGER number)');
+    const distance = parseInt(prompt('>'));
+
+    function calculation() {
+        const moreThan200kmCalculation = distance * PRICE_WITH_DISCOUNT;
+        const lessThan200kmCalculation = distance * PRICE_WITHOUT_DISCOUNT;
```

Comment:
You could do this calculation inside the if and else statements respectively. That way you would not calculate stuff unnecessarily.

Reply:
if(distance > DISTANCE_TO_THE_DISCOUNT) {
            const moreThan200kmCalculation = distance * PRICE_WITH_DISCOUNT;
            console.log(`Your trip will cost $${moreThan200kmCalculation} dollars.`);
            
Like this?

Reply:
Yes